client/src/commands/submitTransaction.ts

```diff
@@ -23,9 +23,18 @@ import { ExtensionCommands } from '../../ExtensionCommands';
 import { VSCodeBlockchainDockerOutputAdapter } from '../logging/VSCodeBlockchainDockerOutputAdapter';
 import { InstantiatedTreeItem } from '../explorer/model/InstantiatedTreeItem';
 
-export async function submitTransaction(treeItem?: InstantiatedTreeItem | TransactionTreeItem): Promise<void> {
+export async function submitTransaction(treeItem?: InstantiatedTreeItem | TransactionTreeItem, evaluate?: boolean): Promise<void> {
```

Comment:
this doesn't need to be optional, you always know if you want to evaluate or submit.